<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergePageResult</class>
 <widget class="QWizardPage" name="MergePageResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Finished</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stage1">
     <property name="text">
      <string>File Created</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stage2">
     <property name="text">
      <string>Text Headers Written</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stage3">
     <property name="text">
      <string>Binary Headers Written</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stage4">
     <property name="text">
      <string>Reading Traces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stage5">
     <property name="text">
      <string>Sorting Traces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stage6">
     <property name="text">
      <string>Writing Traces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="independentegProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="finishSuccessLabel">
     <property name="text">
      <string>File created successfully</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="finishFailureLabel">
     <property name="text">
      <string>&lt;font color='red'&gt;File creation was unsuccessful.&lt;/font&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
